src/helpers.rs

```diff
@@ -895,3 +895,7 @@ impl std::fmt::Display for HexRange {
         write!(f, "[{:#x}..{:#x}]", self.0.start.bytes(), self.0.end().bytes())
     }
 }
+
+pub fn target_os_is_unix(target_os: &str) -> bool {
+    matches!(target_os, "linux" | "macos")
```

Comment:
The target spec contains a `family` field which can contain the `unix` value.

Reply:
https://github.com/rust-lang/miri/pull/2011/files#r889613049